ui/render/sampling.py

```diff
@@ -218,7 +221,3 @@ def draw(self, context):
             col.active = False
         
         col.prop(config, "light_strategy")
-        
-        # Tiled path
-        if config.engine == "PATH":
-            layout.prop(config, "use_tiles")
```

Comment:
Why are you removing this?

Reply:
The option is moved to the normal sampling panel. i.e. line 47 ff.

Reply:
Ah you're right, thanks I missed it